### PR TITLE
fix(web): show a clearer confirmation message when deleting an unnamed album

### PR DESCRIPTION
**Pull request by Snowknight26**

### CODE REVIEW

File: web/src/lib/utils/album-utils.ts [hunk 1]
```diff
@@ -199,3 +200,16 @@ export const collapseAllAlbumGroups = (groupIds: string[]) => {
 export const expandAllAlbumGroups = () => {
   collapseAllAlbumGroups([]);
 };
+
+export const getDeleteAlbumConfirmationDialog = async (album: AlbumResponseDto) => {
```
Comment (jrasm91) [inline web/src/lib/utils/album-utils.ts:204]: Let's rename this to `confirmAlbumDelete()`

**Review by jrasm91 — APPROVED** [review]
Noice

**Comment by pikapower9080** [comment]
Should it be clarified that assets in the album won't be deleted or is that obvious?

**Comment by jrasm91** [comment]
Can you fix the formatting?

**Comment by Snowknight26** [comment]
Doh, thanks for the reminder. Feels like this could be a great opportunity to have a commit hook that auto-formats.

**Review by alextran1502 — APPROVED** [review]

**Pull request merged**